frontend/app/assets/stylesheets/spree/frontend/screen.css.scss

```diff
@@ -995,7 +995,7 @@ div[data-hook="inside_cart_form"] {
 }
 
 .cart-subtotal, .cart-total {
-  background: #00ADEE;
+  background: $link_text_color;
```

Comment:
This should be `$cart-total-color` with a default value of `$link_text_color` so it can be changed independently.